backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/TypeImpl.java

```diff
@@ -194,8 +206,8 @@ public EList<Feature> getFeature() {
      */
     @Override
     public EList<FeatureMembership> getFeatureMembership() {
-        List<FeatureMembership> data = new ArrayList<>();
-        return new EcoreEList.UnmodifiableEList<>(this, SysmlPackage.eINSTANCE.getType_FeatureMembership(), data.size(), data.toArray());
+        FeatureMembership[] data = Stream.concat(getOwnedFeatureMembership().stream(), inheritedMemberships(new BasicEList<Type>()).stream()).toArray(FeatureMembership[]::new);
```

Comment:
Could you rename this variable form `data` to `featureMemberships` please?

Reply:
Done